widget/css/widget.app.css

```diff
@@ -269,5 +269,27 @@ ol li{
   font-size: 16px;
 }
 .person-position {
+  font-size: 12px;
```

Comment:
I’ve requested that font sizes be in em’s or rem’s for all plugins. Any reason why we are not adhering to this?

Reply:
We've updated the font sizing to em's now. We are just making sure it properly renders on multiple devices.